### ISSUE TITLE
Adding a section on signing transactions (for voting on proposals, proving wallet address ownership etc).

### ISSUE THREAD

**Issue by digitalisland** [ethhub-io/ethhub#199]
I can't see any information on how to sign a transaction (for voting, proof of ownership etc) and was going to add one. I am still learning how to make pull requests at this stage though, and to be honest am probably not the best person to explain this procedure in plain English either. It would be good to have a section on this - perhaps in 'Using Ethereum? Thx, Ana.

**Comment by ChazSchmidt** [comment]
I'll add it to the list. Thanks for the suggestion.

**Comment by ChazSchmidt** [comment]
https://github.com/ethhub-io/ethhub/blob/master/docs/using-ethereum/transactions.md#signing-transactions-on-ethereum
